double-check-tmp.md

```diff
@@ -0,0 +1,28 @@
+- No copyright/license mentioned
+  + http://www.jjj.de/fxt/fxtbook.pdf
+  + http://igm.univ-mlv.fr/~mac/REC/text-algorithms.pdf
+  + http://www.ethoberon.ethz.ch/WirthPubl/CBEAll.pdf
+  + http://lampwww.epfl.ch/~schinz/thesis-final-A4.pdf
+  + http://www.stack.nl/~marcov/compiler.pdf
+  + http://arxiv.org/pdf/1206.1754v2.pdf
+  + http://www.csee.umbc.edu/csee/research/cadip/readings/IR.report.120600.book.pdf
+  + https://www.ics.uci.edu/~welling/teaching/ICS273Afall11/IntroMLBook.pdf
+  + http://home.iitk.ac.in/~arlal/book/nptel/pdf/book_linear.pdf
+  + http://www.ii.uib.no/~michal/und/i227/book/book.pdf
+  + http://www.math.cornell.edu/~bterrell/dn.pdf
+  + http://softwarebyrob.wpengine.netdna-cdn.com/assets/Software_by_Rob%20_How_to_Become_a%20_Programmer_1.0.pdf
+  + http://homepages.inf.ed.ac.uk/dts/pm/Papers/nasa-manage.pdf
+  + http://www.tac.mta.ca/tac/reprints/articles/22/tr22.pdf
+- Simply "copyright"
+  + http://algorithmics.lsi.upc.edu/docs/Dasgupta-Papadimitriou-Vazirani.pdf
+  + http://www.ethoberon.ethz.ch/WirthPubl/AD.pdf
+  + http://www.cs.cmu.edu/~rwh/theses/okasaki.pdf
+  + http://www.iro.umontreal.ca/~bengioy/papers/ftml_book.pdf
+  + https://www.math.byu.edu/klkuttle/linearalgebra.pdf
+  + http://hintjens.wdfiles.com/local--files/main%3Afiles/cc1pe.pdf
+  + http://www.di.unipi.it/~ricci/501302.pdf
```

Comment:
Judgement call. This is a 3Com "white paper". Though 3Com has copyright, white papers are typically freely distributed to advance a technology. It doesn't appear as if Prof. Ricci has anything to do with the white paper. The link was likely lost on 3com's acquisition. I suggest replacing it with https://web.archive.org/web/20080401000000*/http://www.3com.com/other/pdfs/infra/corpinfo/en_US/501302.pdf
because if the owner wants to remove it they can do so.